CIP-1856/README.md

```diff
@@ -0,0 +1,149 @@
+          "image": <uri | array>,
+          "mediaType": image/<mime_sub_type>,
+          "description": <string | array>,
+          "files": [{
+            "name": <string>,
+            "mediaType": <mime_type>,
+            "src": "<uri | array>"
+          }],
+    
+          <other properties>
+              
+          "strings": {
+              "de-CH": {
+                "name": <string in Swiss German>,
+                "image": <localized uri for Swiss German | array>,
+                "description": <string in Swiss German | array>
+                 <other localized properties>
+              },
+              "it-IT": {
+                "name": <string in Italian>,
+                "image": <localized uri for Italian German | array>,     
+                "description": <string in Italian | array>
+                <other localized properties>
+              },
+                    
+              <other languages and cultures>
+          }
+        },
+      },
+      "version": <version_id>,
+                  
+      <information about collection>
+      
+      "strings": {
+              "de-CH": {
+                 <localized information about collection in de-CH>
+              },
+              "it-IT": {
+                 <localized information about collection in it-IT>
+              },
+                    
+              <other languages and cultures>
+      }
+  }
+}
+
+```
+
+> **Note**
+> This metadata standard extension is backward compatible and it doesn't affect applications using the current standard. Dapps implementing the proposed extended standard can also default on the legacy values if the localized strings are not available on an asset.
+
+## Code example to access localized properties (TypeScript)
+
+To access the localized strings from the fetched metadata for a native asset, we can simply access the JSON properties from the front end by using the user's selected culture:
+
+```
+
+const response = await fetch(`${<BASE_URL>}/policyId/${<policyId>}`);
+            
+const metadata = response.json();
+const policy = metadata["721"][<policy_id>];
+
+function getPolicyString(policy, key, culture="en") {
+    return policy["strings"][culture][key] 
+        ? policy["strings"][culture][key] 
+        : policy[key]; // default value (not localized)
+}
+
+function getAssetString(policy, asset, key, culture="en") {
+    return policy[asset]["strings"][culture][key] 
+        ? policy[asset]["strings"][culture][key] 
+        : policy[asset][key]; // default value (not localized)
+}
+
+console.log(`Default policy property: ${getPolicyString(policy, <policy_property>)}`);
+console.log(`Localized policy property: ${getPolicyString(policy, <policy_property>, "it-IT")}`);
+console.log(`Default asset name: ${getAssetString(policy, <asset_name>, "name")}`);
+console.log(`Localized asset name: ${getAssetString(policy, <asset_name>, "name", "de-CH")}`);
+         
+```
+     
+## Path to Active

```

Comment:
Thanks for adding this section with the relevant points, but could you add something about who will be implementing this... even if you don't have any specific parties yet who are working on acceptance & implementation?  If you've talked with anyone about the need for this standard, or made some progress yourselves, it would be helpful to reflect that here.

Reply:
@rphair thank you very much for the suggestions and refinements! I'm in contact with Patrick Tobler from NMKR to see if we can implement this together (not confirmed yet). I'm also creating a confidential NFT launch pad on the Oasis Network called geniish.io (the idea is to bridge it to Cardano as soon as it's technically feasible with the sidechain kits), so I came up with the idea in collaboration with my business partner Aaron Schmid and my co-SPO Carolina Isler (LaPetiteADA). I also have 6 years of experience in the translation and copywrite branch, so I believe that web3 will follow the same steps from most web2 giants as localization is the key to a globalized market. I'll add this section as soon as I have confirmed the partnerships.

Reply:
@rphair So NMKR has provided valuable peer feedback and they would like to support this CIP as well. I have included this updates on the proposal.